package.json

```diff
@@ -35,23 +35,24 @@
   },
   "homepage": "https://github.com/googleapis/release-please#readme",
   "devDependencies": {
+    "@octokit/types": "^2.5.1",
     "@types/chai": "^4.1.7",
     "@types/mocha": "^7.0.0",
     "@types/node": "^11.13.6",
     "@types/semver": "^7.0.0",
     "c8": "^7.0.0",
     "chai": "^4.2.0",
     "cross-env": "^7.0.0",
-    "gts": "^1.0.0",
+    "gts": "^2.0.0-alpha.8",
```

Comment:
I saw some of package use `next` tag here. Dose it matter here?

Reply:
I prefer being explicit with the dependency, as soon as we do a few GAPIC library updates, I think we can go ahead and migrate gts to 2.x 😄